aries_vcx/src/handlers/issuance/issuer.rs

```diff
@@ -209,6 +209,20 @@ impl Issuer {
         self.issuer_sm.find_message_to_handle(messages)
     }
 
+    pub fn get_revocation_id(&self) -> VcxResult<String> {
+        self.issuer_sm
+            .get_revocation_info()
+            .ok_or(AriesVcxError::from_msg(
+                AriesVcxErrorKind::InvalidState,
+                "Credential is has not yet been created",
+            ))?
+            .cred_rev_id
```

Comment:
Tangential to the PR, but `cred_rev_id` probably shouldn't be a string?

Reply:
yeah, but that issue roots from vdrtools

Reply:
OK, I apologize for the comment, then.